mgmt_cli_test.py

```diff
@@ -121,8 +136,22 @@ def create_ks_and_tables(self, num_ks, num_table):
                 # can use this function to populate tables better?
                 # self.populate_data_parallel()
 
-    def verify_backup_success(self):
-        pass
+    # pylint: disable=too-many-arguments
+    def verify_backup_success(self, bucket, cluster_id, keyspace_name='keyspace1', table_name='standard1',
+                              local='/tmp/backup', truncate=True):
+        self.download_file_from_backup_repo(local=local, bucket=bucket, cluster_id=cluster_id)
```

Comment:
maybe a comment on why this is a temporary thing, i.e. until manager would support it

Reply:
It is, for now, our mechanism to recover the backed up data to the cluster, to verify it's data (data corruption verification) for backup tests.
It could be afterwords moved to some mgmt.py or some utils/tools.